modules/lex_calendar/templates/calendar-page.html.twig

```diff
@@ -54,25 +61,67 @@ window.addEventListener( 'load', function(){
       },
       eventAfterAllRender: function() {
         var view = $('#calendar').fullCalendar('getView');
-
+        var relatedDepts = [];
         // reset calendar dots
         $('#calendar').find('.fc-event').remove();
 
         // manage dots
         var events = $('#calendar').fullCalendar('clientEvents');
 
         for (var i = 0; i < events.length; i++) {
```

Comment:
Can we use a map function here? Keeps you from having to do `var event = events[i]`